### PR TITLE
adding requirements asked here: https://github.com/geosolutions-it/De…

### PR DESCRIPTION
**Pull request by lpasquali**
…vOps/issues/391

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -49,23 +53,29 @@ RUN mkdir -p \
...
+  -Dorg.geotools.coverage.jaiext.enabled=${JAIEXT_ENABLED} \
   -Duser.timezone=GMT \
   -Dorg.geotools.shapefile.datetime=true \
   -DGEOSERVER_LOG_LOCATION=${GEOSERVER_LOG_LOCATION} \
   -DGEOWEBCACHE_CONFIG_DIR=${GEOWEBCACHE_CONFIG_DIR} \
   -DGEOWEBCACHE_CACHE_DIR=${GEOWEBCACHE_CACHE_DIR} \
   -DNETCDF_DATA_DIR=${NETCDF_DATA_DIR} \
-  -DGRIB_CACHE_DIR=${GRIB_CACHE_DIR}"
+  -DGRIB_CACHE_DIR=${GRIB_CACHE_DIR} \
+  ${EXTRA_OPT}"
```
Comment (randomorder) [inline Dockerfile:78]: supposed to be EXTRA_OPT**S**

File: Dockerfile [hunk 2]
```diff
@@ -49,23 +53,29 @@ RUN mkdir -p \
     "${NETCDF_DATA_DIR}" \
     "${GRIB_CACHE_DIR}"
 
+
 # copy from mother
 COPY --from=mother "${GEOSERVER_DATA_DIR}" "${GEOSERVER_DATA_DIR}"
 COPY --from=mother "${CATALINA_BASE}/webapps" "${CATALINA_BASE}/webapps"
 
+#install needed packages
+RUN apt-get update && apt-get install --yes gdal-bin pgcli fontconfig libfreetype6
```
Comment (randomorder) [inline Dockerfile:62]: use postgresql-client instead og pgcli please

**Review by randomorder — CHANGES_REQUESTED** [review]
looks good please fix the typo and swap the postgresql client application

**Comment by randomorder** [comment]
Please also switch base image like I did [here](https://github.com/geosolutions-it/docker-geoserver/blob/docker_hub_build/Dockerfile#L1)

Because of this: 
image

**Comment by lpasquali** [comment]
latest commits should fix everything you requested @randomorder

**Review by randomorder — APPROVED** [review]

**Pull request merged**